src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,20 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        int residue;
+        String result = "";
+        StringBuffer container = new StringBuffer();
```

Comment:
read [checklist](https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_01.html) attentively
- Don’t complicate if-else construction: example
- Use StringBuilder instead of String or StringBuffer.

Reply:
done, however i can't write without construction if, because we should check variable about zero.